package.json

```diff
@@ -34,17 +34,15 @@
     "url": "https://github.com/MetaMask/eth-ledger-bridge-keyring/issues"
   },
   "homepage": "https://github.com/MetaMask/eth-ledger-bridge-keyring#readme",
-  "resolutions": {
-    "mocha/**/mkdirp": "^0.5.3"
```

Comment:
I ran `yarn npm audit` locally to confirm this was no longer needed. No advisories :+1: